src/csda_client/models.py

```diff
@@ -13,7 +14,7 @@ class QuotaUnit(Enum):
 
 class Profile(BaseModel):
     earthdata_username: str
-    title: str
+    title: Optional[str] = None
```

Comment:
This is the "more modern" way:

```suggestion
    title: str | None = None
```